src/index.js

```diff
@@ -26,6 +26,7 @@ export {
 } from './themes/index.js';
 export {default as LocaleProvider} from './locale/index.js';
 export {default as BaseProvider} from './helpers/base-provider.js';
+export {getMediaQueries, getMediaQuery} from './helpers/responsive-helpers.js';
```

Comment:
To be honest I'm on a fence regarding including it as a part of `baseui` API. 
Also, if we do think there is a value in providing `mediaQuery` maybe we should combine and ship all the helpers from `baseui/helpers` or `baseui/utils`.

Reply:
Reverted exposing the helper to the API